README.md

```diff
@@ -55,15 +55,15 @@ If you’d like to offer feedback, come across any problems, or have any questio
 
 ### 2.0. Initial considerations<a id="2_0"></a>
 
->A **batch** is a rollup block
+>A **batch** is a rollup "block" 
```

Comment:
```suggestion
>A **batch** is a rollup block
```

Reply:
not a huge fan of quotation marks around words. i think it's clear enough without